packages/webview_flutter/tizen/src/webview.h

```diff
@@ -0,0 +1,69 @@
+#ifndef FLUTTER_PLUGIN_WEBVIEW_FLUTTER_TIZEN_WEVIEW_H_
+#define FLUTTER_PLUGIN_WEBVIEW_FLUTTER_TIZEN_WEVIEW_H_
+
+#include <flutter/method_channel.h>
+#include <flutter/plugin_registrar.h>
+#include <flutter/standard_message_codec.h>
+#include <flutter/standard_method_codec.h>
+#include <flutter_platform_view.h>
+#include <flutter_tizen_texture_registrar.h>
+#include <tbm_surface.h>
+
+namespace LWE {
+class WebContainer;
+}
+
+class TextInputChannel;
+
+class WebView : public PlatformView {
+ public:
+  WebView(flutter::PluginRegistrar* registrar, int viewId,
+          FlutterTextureRegistrar* textureRegistrar, double width,
+          double height, flutter::EncodableMap& params);
+  ~WebView();
+  virtual void Dispose() override;
+  virtual void Resize(double width, double height) override;
+  virtual void Touch(int type, int button, double x, double y, double dx,
+                     double dy) override;
+  virtual void SetDirection(int direction) override;
+  virtual void ClearFocus() override;
+
+  // Key input event
+  virtual void DispatchKeyDownEvent(Ecore_Event_Key* key) override;
+  virtual void DispatchKeyUpEvent(Ecore_Event_Key* key) override;
+  virtual void DispatchCompositionUpdateEvent(const char* str,
+                                              int size) override;
+  virtual void DispatchCompositionEndEvent(const char* str, int size) override;
+
```

Comment:
When I tried to run example on Tizen device (`flutter-tizen run -v`), two errors occurred:

```
           "/home/r.walczyna/tizen-studio/tools/smart-build-interface/../llvm-10/bin/clang++" -c "src/webview_flutter_tizen_plugin.cc" -o "Debug/src/webview_flutter_tizen_plugin.o" -DTIZEN_DEPRECATION -DDEPRECATION_WARNING
-DTIZEN_DEPRECATION
[...]
           In file included from src/webview_flutter_tizen_plugin.cc:16:
           src/webview.h:34:16: error: 'DispatchCompositionUpdateEvent' marked 'override' but does not override any member functions
             virtual void DispatchCompositionUpdateEvent(const char* str,
                          ^
           src/webview.h:36:16: error: 'DispatchCompositionEndEvent' marked 'override' but does not override any member functions
             virtual void DispatchCompositionEndEvent(const char* str, int size) override;
                          ^
           2 errors generated.
```

After removing not needed `overrides` it has built successful.

Reply:
It seems that you need to update flutter-tizen. Actually, I have updated "flutter_platform_view.h" in flutter engine and the file should been applied on flutter-tizen cache side.

Reply:
An engine release containing [this patch](https://github.com/flutter-tizen/engine/pull/52) has just been released yesterday. The embedder API versioning is a quite complicated problem because a plugin can have only a constraint on the Flutter SDK version (below) but not the Tizen embedder version. I'll tell you a bit more about this matter later.

```yaml
environment:
  flutter: ">=2.0.1"
```